sdk/src/client/secret/ledger_nano.rs

```diff
@@ -166,6 +169,16 @@ impl SecretManage for LedgerSecretManager {
         Ok(ed25519_addresses)
     }
 
+    async fn generate_evm_addresses(
+        &self,
+        _coin_type: u32,
+        _account_index: u32,
+        _address_indexes: Range<u32>,
+        _options: Option<GenerateAddressOptions>,
+    ) -> Result<Vec<EvmAddress>, Self::Error> {
+        todo!()
```

Comment:
The iota/shimmer ledger apps don't support it, one should use the ethereum app instead, so I would say it should just return an error that it's not supported here